### DISCUSSION TITLE
Calculations

### DISCUSSION THREAD

**Discussion by karpadiem** [OpnForm/OpnForm#144] [Ideas & Feature Requests]
Really excited about OpnForm. Seems to pretty much fill the void we are trying to fill for forms. One feature we are missing is some sort of calculation based on options selected.

Is this something we could add ourselves using javascript and the custom code field for a form? If so, is there any documentation you could point us in the direction of?

**Comment by JhumanJ** [comment]
Closing in favor of https://feedback.opnform.com/p/computed-fields
